app.go

```diff
@@ -404,16 +417,16 @@ func HandleAction(action interface{}, context *Context) error {
 	if len(vals) == 0 {
 		fmt.Fprintln(os.Stderr,
 			"DEPRECATED Action signature.  Must be `cli.ActionFunc`")
-		return nil
+		return err
```

Comment:
I kinda like `return nil` here -- `return err` visually implies to me that it it is not `nil`.